new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java

```diff
@@ -0,0 +1,655 @@
+            .name(secretName)
+            .namespace(namespace))
+        .stringData(secretMap)), "Create secret failed with ApiException");
+    assertTrue(secretCreated, String.format("create secret failed for %s", secretName));
+  }
+
+  /** Scale the WebLogic cluster to specified number of servers.
+   *  And verify the sample app can be accessed through NGINX.
+   *
+   * @param clusterName the WebLogic cluster name in the domain to be scaled
+   * @param domainUid the domain to which the cluster belongs
+   * @param domainNamespace the domain namespace in which the domain exists
+   * @param replicasBeforeScale the replicas of the WebLogic cluster before the scale
+   * @param replicasAfterScale the replicas of the WebLogic cluster after the scale
+   * @param curlCmd the curl command to verify ingress controller can access the sample apps from all managed servers
+   *                in the cluster, if curlCmd is null, the method will not verify the accessibility of the sample app
+   *                through ingress controller
+   * @param expectedServerNames list of managed servers in the cluster before scale, if curlCmd is null,
+   *                            set expectedServerNames to null too
+   */
+  public static void scaleAndVerifyCluster(String clusterName,
+                                           String domainUid,
+                                           String domainNamespace,
+                                           int replicasBeforeScale,
+                                           int replicasAfterScale,
+                                           String curlCmd,
+                                           List<String> expectedServerNames) {
+
+    String manageServerPodNamePrefix = domainUid + "-" + clusterName + "-" + MANAGED_SERVER_NAME_BASE;
+
+    // get the original managed server pod creation timestamp before scale
+    List<String> listOfPodCreationTimestamp = new ArrayList<>();
+    for (int i = 1; i <= replicasBeforeScale; i++) {
+      String managedServerPodName = manageServerPodNamePrefix + i;
+      String originalCreationTimestamp =
+          assertDoesNotThrow(() -> getPodCreationTimestamp(domainNamespace, "", managedServerPodName),
+              String.format("getPodCreationTimestamp failed with ApiException for pod %s in namespace %s",
+                  managedServerPodName, domainNamespace));
+      listOfPodCreationTimestamp.add(originalCreationTimestamp);
+    }
+
+    // scale the cluster in the domain
+    logger.info("Scaling cluster {0} of domain {1} in namespace {2} to {3} servers",
+        clusterName, domainUid, domainNamespace, replicasAfterScale);
+    assertThat(assertDoesNotThrow(() -> scaleCluster(domainUid, domainNamespace, clusterName, replicasAfterScale)))
+        .as("Verify scaling cluster {0} of domain {1} in namespace {2} succeeds",
+            clusterName, domainUid, domainNamespace)
+        .withFailMessage("Scaling cluster failed")
```

Comment:
include more detail, like you do in the as()

Reply:
fixed